paper.tex

```diff
@@ -1037,6 +1057,8 @@ \section*{Discussion}
 point for other research. We provide an efficient algorithm and 
 reference implementation]
 
+[A paragraph about another major sort of uncertainty: polytomies]
```

Comment:
mostly we've talked about the missing information about where recomb nodes go; but missing coal nodes inducing polytomies is an other important thing.